### PR TITLE
ETFE-2948 Alignment - Category of Wine

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: test/navigation/ItemsNavigatorSpec.scala [hunk 1]
```diff
@@ -457,13 +458,13 @@ class ItemsNavigatorSpec extends SpecBase with ItemFixtures {
           }
         }
 
-        "to the Imported Wine Choice page" - {
+        "to the ImportedWine Wine Choice page" - {
```
Comment (grahampaulcook) [inline test/navigation/ItemsNavigatorSpec.scala:461]:
```suggestion
        "to the ItemWineProductCategoryPage" - {
```

File: test/navigation/ItemsNavigatorSpec.scala [hunk 2]
```diff
@@ -493,13 +494,13 @@ class ItemsNavigatorSpec extends SpecBase with ItemFixtures {
...
```
Comment (grahampaulcook) [inline test/navigation/ItemsNavigatorSpec.scala:497]:
```suggestion
        "to the ItemWineProductCategoryPage" - {
```

File: test/navigation/ItemsNavigatorSpec.scala [hunk 3]
```diff
@@ -743,18 +744,18 @@ class ItemsNavigatorSpec extends SpecBase with ItemFixtures {
...
-              .set(ItemImportedWineFromEuChoicePage(testIndex1), false)
+              .set(ItemWineProductCategoryPage(testIndex1), ImportedWine)
             ) mustBe itemsRoutes.ItemWineOriginController.onPageLoad(testErn, testDraftId, testIndex1, NormalMode)
           }
         }
       }
 
       "must go from the ItemWineOperationsChoicePage" - {
-        "to the Imported Wine Choice page" in {
+        "to the ImportedWine Wine Choice page" in {
```
Comment (grahampaulcook) [inline test/navigation/ItemsNavigatorSpec.scala:756]:
```suggestion
        "to the ItemWineProductCategoryPage" in {
```

File: test/pages/sections/items/ItemsSectionItemSpec.scala [hunk 4]
```diff
@@ -36,7 +37,7 @@ class ItemsSectionItemSpec extends SpecBase with ItemFixtures {
 
     "must return true" - {
 
-      "when Bulk Wine, Imported from EU, with all mandatory and optional pages complete" in {
+      "when Bulk Wine, ImportedWine from EU, with all mandatory and optional pages complete" in {
```
Comment (grahampaulcook) [inline test/pages/sections/items/ItemsSectionItemSpec.scala:40]:
```suggestion
      "when Bulk Wine, Imported from EU, with all mandatory and optional pages complete" in {
```

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢 🚀

**Pull request merged**